_layouts/default.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
     {% include head.html %}
+    <link rel="stylesheet" href="/css/info-block.css">
```

Comment:
I'd probably move this to `head.html` include. Although we have `link`s here as well.

Reply:
I agree - this is the better place for css that might need to be included anywhere in the project.